ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml

```diff
@@ -2240,6 +2264,9 @@ keys:
           '
       users:
```

Comment:
Any reason to not merge this with the underlying `eos_cli_config_gen` schema for user?
I can see that probably `localized` could lead to possible confusion (which one has precedence over which) but it will be probably simpler to maintain in the long run.

It would be bringing `remote_address` and `udp_port` in `eos_designs`
NOTE: valid values in `eos_designs` schema for algorithm are better than the underlying ones
NOTE2: it would be necessary to maintain different descriptions in `eos_designs`

Reply:
Because of your notes + the fact that we do not support generating localized ids for remote engine ids.